docs/README.md

```diff
@@ -0,0 +1,9 @@
+# Aragon Network Token
+
+Welcome to the technical documentation for the Aragon Network Token (ticker: [ANT](https://coinmarketcap.com/currencies/aragon/)).
+
+- The [**Token**](token/behaviour.md) section contains technical information about the token itself
+- The [**Developers**](developers/getting-started.md) section contains useful information for developers wishing to programmatically interact with the token or initial sale
```

Comment:
If possible I'd rename these files to `readme.md` and reference in there the other files stored in each of these dirs, it improves a lot the UX when browsing the files

Reply:
I'll add READMEs to each inner dir to serve as a table of contents, but the structure here is primarily intended to be consumed as a gitbook. Once this is merged, docs.aragon.org/ant will have these docs :).